docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>technically h5py could be used with fsspec too, but I think it's fine to do it this way on downloaded data. As noted above this should be a fall back method if the Xarray/S3 doesn't meet the user need.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>